client/components/language-picker/language-names.js

```diff
@@ -0,0 +1,141 @@
+export default {
+	/* eslint-disable quote-props */
+	'af': translate => translate( 'Afrikaans' ),
+	'als': translate => translate( 'Alemannic' ),
+	'am': translate => translate( 'Amharic' ),
+	'ar': translate => translate( 'Arabic' ),
+	'arc': translate => translate( 'Aramaic' ),
+	'as': translate => translate( 'Assamese' ),
+	'ast': translate => translate( 'Asturian' ),
+	'av': translate => translate( 'Avaric' ),
+	'ay': translate => translate( 'Aymara' ),
+	'az': translate => translate( 'Azerbaijani' ),
+	'ba': translate => translate( 'Bashkir' ),
+	'bel': translate => translate( 'Belarusian' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Belarus' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).